core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -505,7 +525,7 @@ public String buildKeyName(long keynum) {
     if (!orderedinserts) {
       keynum = Utils.hash(keynum);
     }
-    return "user" + keynum;
+	return "user"+String.format("%0"+zeropadding+"d",keynum);
```

Comment:
I'm new to Github/coding so I don't know how to make this comment look at the latest code I pushed to the branch, but there's the latest function that addresses performance concerns
public String buildKeyName(long keynum) {
    if (!orderedinserts) {
      keynum = Utils.hash(keynum);
    }
	String value=Long.toString(keynum);
	int fill=zeropadding - value.length();
	String prekey="user";
	for(int i=0; i<fill;i++) {prekey+='0';}
	return prekey+value;
  }